hotline/species/indexes.py

```diff
@@ -0,0 +1,19 @@
+from elasticmodels import Index, StringField
+
+from .models import Species
+
+
+class SpeciesIndex(Index):
+
+    severity = StringField(attr="get_severity")
+    category = StringField(attr="get_category")
+
```

Comment:
This isn't in the Elasticmodels docs, but to improve performance, you can override get_queryset so it includes the severity and category:
```
def get_queryset(self):
    return super.get_queryset().select_related("severity", "category")
```